cleaned_data/SpanishLanguageRecordings/rfta:171.xml

```diff
@@ -0,0 +1,400 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mods xmlns="http://www.loc.gov/mods/v3"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      xmlns:xs="http://www.w3.org/2001/XMLSchema"
+      xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+   <identifier type="local">20220119_Duarte_Mitsy-Gissela</identifier>
+   <identifier type="pid">rfta:171</identifier>
+   <titleInfo>
+      <title>Interview with Mitsy Gissela Duarte, 2022-01-19</title>
+   </titleInfo>
+   <titleInfo lang="spa">
+      <title>Entrevista con Mitsy Gissela Duarte, 2022-01-19</title>
+   </titleInfo>
+   <abstract>In this interview Mitsy Gissela Duarte, a Honduran immigrant and Gatlinburg resident since 2003, tells us about her experience of awakening to a city full of smoke and fire, fleeing alongside other residents and the deep sadness that ensued with the return to the city.</abstract>
```

Comment:
For the abstracts, can you include the representative quote at the end prefaced by the text "This is a representative quote from the interview:" (for English abstract) or "Esta es una cita representativa de la entrevista:" (for Spanish abstract)?